test/replaceScopeName.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const replaceScopeName = require('../src/replaceScopeName')
+const test = require('ava')
+
+const scenarios = [
+  {
+    testName: 'Return empty string if none available',
+    expectedName: ''
+  },
+  {
+    testName: 'Return same name if not scoped',
+    name: 'poopie',
+    expectedName: 'poopie'
+  },
+  {
+    testName: 'Scoped name with default divider',
+    name: '@poopie/core',
+    expectedName: 'poopie_core'
+  },
+  {
+    testName: 'Scoped name using a custom divider',
+    name: '@poopie/core',
+    divider: '-',
+    expectedName: 'poopie-core'
+  }
+]
+
+for (const scenario of scenarios) {
+  test(scenario.testName, t => t.is(replaceScopeName(scenario.name, scenario.divider), scenario.expectedName))
+}
```

Comment:
I think I want to get rid of this pattern from here & from `test/getHomepage.js` because it's harder to run a single test.

Reply:
Let me know what you end up choosing and I'll change it here too